<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>539</width>
    <height>486</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Урок 06</string>
  </property>
  <property name="locale">
   <locale language="Russian" country="Russia"/>
  </property>
  <property name="inputMethodHints">
   <set>Qt::ImhNone</set>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>7</x>
      <y>250</y>
      <width>521</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>30</y>
      <width>151</width>
      <height>211</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QPushButton" name="new_button">
       <property name="text">
        <string>Новый</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="open_button">
       <property name="locale">
        <locale language="Russian" country="Russia"/>
       </property>
       <property name="text">
        <string>Открыть</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="open_read_button">
       <property name="locale">
        <locale language="Russian" country="Russia"/>
       </property>
       <property name="text">
        <string>Открыть на чтение</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="save_button">
       <property name="locale">
        <locale language="Russian" country="Russia"/>
       </property>
       <property name="text">
        <string>Сохранить</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="print_button">
       <property name="locale">
        <locale language="Russian" country="Russia"/>
       </property>
       <property name="text">
        <string>Печать</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="help_button">
       <property name="locale">
        <locale language="Russian" country="Russia"/>
       </property>
       <property name="text">
        <string>Справка</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="settings_label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>273</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Настройки</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="hotkeys_group">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>370</y>
      <width>481</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string>Горячие клавиши</string>
    </property>
    <widget class="Line" name="line_2">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>30</y>
       <width>16</width>
       <height>51</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget_7">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>30</y>
       <width>211</width>
       <height>23</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QLabel" name="label_new">
        <property name="text">
         <string>Новый</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_5">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLineEdit" name="new_edit">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="mouseTracking">
         <bool>true</bool>
        </property>
        <property name="focusPolicy">
         <enum>Qt::StrongFocus</enum>
        </property>
        <property name="locale">
         <locale language="English" country="UnitedStates"/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>211</width>
       <height>23</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="label_open">
        <property name="text">
         <string>Открыть</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLineEdit" name="open_edit">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="mouseTracking">
         <bool>true</bool>
        </property>
        <property name="focusPolicy">
         <enum>Qt::StrongFocus</enum>
        </property>
        <property name="contextMenuPolicy">
         <enum>Qt::NoContextMenu</enum>
        </property>
        <property name="acceptDrops">
         <bool>false</bool>
        </property>
        <property name="whatsThis">
         <string notr="true"/>
        </property>
        <property name="accessibleName">
         <string notr="true"/>
        </property>
        <property name="accessibleDescription">
         <string notr="true"/>
        </property>
        <property name="locale">
         <locale language="English" country="UnitedStates"/>
        </property>
        <property name="inputMethodHints">
         <set>Qt::ImhNone</set>
        </property>
        <property name="inputMask">
         <string notr="true"/>
        </property>
        <property name="maxLength">
         <number>32767</number>
        </property>
        <property name="frame">
         <bool>true</bool>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::Normal</enum>
        </property>
        <property name="readOnly">
         <bool>false</bool>
        </property>
        <property name="placeholderText">
         <string notr="true"/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>212</width>
       <height>23</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLabel" name="label_save">
        <property name="text">
         <string>Сохранить</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLineEdit" name="save_edit">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="mouseTracking">
         <bool>true</bool>
        </property>
        <property name="focusPolicy">
         <enum>Qt::StrongFocus</enum>
        </property>
        <property name="locale">
         <locale language="English" country="UnitedStates"/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget_8">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>60</y>
       <width>211</width>
       <height>23</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_8">
      <item>
       <widget class="QLabel" name="label_quit">
        <property name="text">
         <string>Выход</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_6">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLineEdit" name="quit_edit">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="mouseTracking">
         <bool>true</bool>
        </property>
        <property name="focusPolicy">
         <enum>Qt::StrongFocus</enum>
        </property>
        <property name="locale">
         <locale language="English" country="UnitedStates"/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="lang_group">
    <property name="geometry">
     <rect>
      <x>31</x>
      <y>302</y>
      <width>201</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string>Язык</string>
    </property>
    <widget class="QRadioButton" name="english_radio_button">
     <property name="geometry">
      <rect>
       <x>14</x>
       <y>24</y>
       <width>69</width>
       <height>19</height>
      </rect>
     </property>
     <property name="locale">
      <locale language="English" country="UnitedStates"/>
     </property>
     <property name="text">
      <string>English</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="russian_radio_button">
     <property name="geometry">
      <rect>
       <x>109</x>
       <y>24</y>
       <width>77</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Русский</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="theme_group">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>302</y>
      <width>201</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string>Оформление</string>
    </property>
    <widget class="QRadioButton" name="light_radio_button">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>24</y>
       <width>77</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Светлое</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="dark_radio_button">
     <property name="geometry">
      <rect>
       <x>115</x>
       <y>24</y>
       <width>72</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Тёмное</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>539</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="task_5_3_menu">
    <property name="locale">
     <locale language="Russian" country="Russia"/>
    </property>
    <property name="title">
     <string>Задание 5.3</string>
    </property>
    <addaction name="finder_action"/>
   </widget>
   <widget class="QMenu" name="file_menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="open_action"/>
    <addaction name="open_rd_action"/>
    <addaction name="save_action"/>
    <addaction name="separator"/>
    <addaction name="print_action"/>
    <addaction name="separator"/>
    <addaction name="help_action"/>
   </widget>
   <widget class="QMenu" name="edit_menu">
    <property name="title">
     <string>Правка</string>
    </property>
    <addaction name="copy_action"/>
    <addaction name="cut_action"/>
    <addaction name="paste_action"/>
   </widget>
   <addaction name="file_menu"/>
   <addaction name="edit_menu"/>
   <addaction name="task_5_3_menu"/>
  </widget>
  <action name="finder_action">
   <property name="text">
    <string>Просмотрщик</string>
   </property>
  </action>
  <action name="open_action">
   <property name="text">
    <string>Открыть</string>
   </property>
  </action>
  <action name="open_rd_action">
   <property name="text">
    <string>Открыть на чтение</string>
   </property>
  </action>
  <action name="save_action">
   <property name="text">
    <string>Сохранить</string>
   </property>
  </action>
  <action name="help_action">
   <property name="text">
    <string>Справка</string>
   </property>
  </action>
  <action name="copy_action">
   <property name="text">
    <string>Копировать</string>
   </property>
  </action>
  <action name="cut_action">
   <property name="text">
    <string>Вырезать</string>
   </property>
  </action>
  <action name="paste_action">
   <property name="text">
    <string>Вставить</string>
   </property>
  </action>
  <action name="print_action">
   <property name="text">
    <string>Печать</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
